assets/src/application/ui/layout/espressoTable/table-body.js

```diff
@@ -50,114 +50,93 @@ const TableBody = ( {
 	 * @param {Object} cellData
 	 * @return {Object} rendered headings row
 	 */
-	const tableCell = ( rowNumber, colNumber, column, cellData ) => {
+	const tableCell = (rowNumber, colNumber, column, cellData) => {
 		return hasRowHeaders && colNumber === 0 ? (
 			<TableHeaderCell
-				key={ `row-${ rowNumber }-col-${ colNumber }` }
-				rowNumber={ rowNumber }
-				colNumber={ colNumber }
-				rowType={ 'body' }
-				htmlId={ cellData.id || tableId }
-				htmlClass={ cellData.class || '' }
-				cssClasses={ cssClasses }
+				key={`row-${rowNumber}-col-${colNumber}`}
+				rowNumber={rowNumber}
+				colNumber={colNumber}
+				rowType={'body'}
+				htmlId={cellData.id || tableId}
+				htmlClass={cellData.class || ''}
```

Comment:
Where does `cellData` come from? since `class` is a keyword, it's better to use `className` to make destructuring easy.

Reply:
Dunno where it comes from. @tn3rb can help here, maybe.
***
Updated: https://github.com/eventespresso/event-espresso-core/pull/2365/commits/b2d81548b9f038c0961fc3043545bbc78955fbc7